boxsdk/object/api_json_object.py

```diff
@@ -2,12 +2,17 @@
 
 from __future__ import unicode_literals, absolute_import
 
-from collections import Mapping
+import sys
 from abc import ABCMeta
 
 from .base_api_json_object import BaseAPIJSONObject, BaseAPIJSONObjectMeta
 from ..util.compat import with_metaclass
 
+if sys.version_info >= (3, 3):
```

Comment:
Using this style rather than `try/except` due to the reasoning I outlined in this post: https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/